docs/self-hosted-appcircle/self-hosted-runner/runner-vm-setup.md

```diff
@@ -510,10 +510,18 @@ tart clone macOS_240514 vm01
   </TabItem>
 </Tabs>
 
+:::tip
+It's not recommended to delete the base image (`macOS_240514`) as it won't save disk space due to copy-on-write file system on macOS. You can safely re-create `vm01` from the same base image `macOS_240514` without downloading and extracting again from network if needed. 
```

Comment:
We may generalize the base image name like `macOS_YYMMDD`, `macOS_<version-code>`. Or indicating it is an example: 
> It's not recommended to delete the base image (e.g., `macOS_240514`) as it won't save disk space due to copy-on-write file system on macOS. You can safely re-create `vm01` from the same base image (e.g., `macOS_240514`) without downloading and extracting again from network if needed.